### DISCUSSION TITLE
汇报一个现象，恳请帮助

### DISCUSSION THREAD

**Discussion by lcjssg** [usememos/memos#250] [Ideas]
我的系统是unraid  6.10.3，以docker使用memos，也借用了discussion中的ios快捷方式进行记录，感觉memos非常方便好用。之前一直使用的应该是0.3.1的版本，今天在chrome浏览器中录入一条带图的内容后，发现浏览器端打开memos即显示一片空白，反复刷新并不能有任何显示，重启容器也没有反应，但ios端能正常看到内容。本着更新版本的想法，备份memos_prod.db后将memos更新到最新版本，更新完后将memos_prod.db拷回原文件夹，启动容器后提示登录，登录原账号即提示以下内容：

Unhandled Thrown Error!
Cannot read properties of undefined (reading 'count')
TypeError: Cannot read properties of undefined (reading 'count')
    at http://****/assets/index.f618fa86.js:107:19652
    at Qs (http://****/assets/index.f618fa86.js:32:24425)
    at ci (http://****/assets/index.f618fa86.js:32:42666)
    at S9 (http://****/assets/index.f618fa86.js:32:41472)
    at so (http://****/assets/index.f618fa86.js:32:40498)
    at ty (http://****/assets/index.f618fa86.js:32:37079)
    at Xn (http://****/assets/index.f618fa86.js:30:3274)
    at http://****/assets/index.f618fa86.js:32:34432
💿 Hey developer 👋

You can provide a way better UX than this when your app throws errors by providing your own errorElement props on <Route>

****是我绑定公网ip的域名和端口，后来我通过修改安装路径，搞定了安装，但原来的数据可能是存在一个相对路径中，找不到了。
所以，能增加一个导出、导入的功能吗？作为一个小白，我发现我之前备份的文件好像都是错误的……被自己蠢哭

**Comment by boojack** [comment]
和这个问题有点像 https://github.com/usememos/memos/issues/243

有空的话麻烦通过 telegram 直接私聊我一下

**Reply by lcjssg** [reply]
后面我修改了绝对路径，重新安装了最新的版本后问题消失了，但原来的记录已经找不到了。。。。我没有搞得太明白原来那个相对路径里的memos_prod.db究竟放在哪里了。。。我是直接用 docker run -d --name memos -p 5230:5230 -v ~/.memos/:/var/opt/memos neosmemo/memos:latest 的命令完成安装的，但在unraid的文件夹中找到的memos_prod.db一直都不是真正的文件，怎么写内容文件大小都不变。

**Reply by boojack** [reply]
在你当前登录用户的 home 目录下，类似于这样：
<img width="167" alt="image" src="https://user-images.githubusercontent.com/24653555/193258980-e47175d7-fbdb-40ce-bd98-6e5295d90234.png">

**Reply by boojack** [reply]
> 本着更新版本的想法，备份memos_prod.db后将memos更新到最新版本，更新完后将memos_prod.db拷回原文件夹

更新版本后不要将之前的数据库文件放回去。（之后我会添加一个更新版本的 guide）

**Reply by lcjssg** [reply]
有导入导出功能的话，感觉会挺棒！！

**Reply by lcjssg** [reply]
> 在你当前登录用户的 home 目录下，类似于这样： <img alt="image" width="167" src="https://user-images.githubusercontent.com/24653555/193258980-e47175d7-fbdb-40ce-bd98-6e5295d90234.png">

额，前面提到了我是用unraid系统，docker安装的memos，我没找到这路径…（捂脸）
